include/netlink/route/link/bridge_info.h

```diff
@@ -8,6 +8,7 @@
 
 #include <netlink/netlink.h>
 #include <netlink/route/link.h>
+#include <linux/if_bridge.h>
```

Comment:
This probably won't fly. Are there any other solutions other than just using an int instead of the enum?

Reply:
Why do you say it won't fly?

I didn't previously realize that `BR_BOOLOPT_NO_LL_LEARN` is an enum. Often enough, such identifiers are just int (literals) in kernel header.

I don't think we should/need to use `enum br_boolopt_id`. We also avoid enums in our public API, because their size depends on compiler flags like `-fshort-enums` (not sure that's a very convincing argument).

I think just use a plain `int id`. Then also don't include `<linux/if_bridge.h>`. Let the user include that to get the enum values they need.

Reply:
> Why do you say it won't fly?

Probably because we would be including our copy of the kernel header which would inflate the header size. Also the included kernel header may be older or newer than the one on the user's system, which could be undesirable.

Changing to use an int instead.

Reply:
>  It would use the kernel header that the user has on their system, not the one included in libnl.

Right. The one in libnl under `include/linux-private/` is only to build libnl itself. Those headers are not for the users of the library.

> That means you can't rely on the kernel header file defining a symbol that is used in the libnl header file.

That probably would be fine. We rely on kernel headers to be stable and not break API. In this case, `enum br_boolopt_id` was added in linux v5.0 (March 2019), so if we use the symbol, we would add a dependency on 5.0+ (which is old enough to be OK).

However, I still think using a plain "int" is preferable. Users who want to use this API should include `<linux/if_bridge.h>` themselves and use enums like `BR_BOOLOPT_NO_LL_LEARN`.